qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -541,38 +540,35 @@ def _clean_validate_template(cls, md_template, study_id, restriction_dict,
         # we don't modify the user one
         md_template = md_template.copy(deep=True)
 
+        # In the database, all the column headers are lowercase
+        md_template.columns = [c.lower() for c in md_template.columns]
+        # validating pgsql reserved words not to be column headers
+        current_headers = set(md_template.columns.values)
+        reserved_words = qdb.metadata_template.util.get_pgsql_reserved_words()
+        overlap = reserved_words & current_headers
+        if overlap:
```

Comment:
If `get_invalid_column_names` is only used here I think we can make it return the error message (if any) instead of the list of invalid columns. @ElDeveloper what do you think?